.github/workflows/nightly_release.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       branch: ""
       pip_path: https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
       pre_dev_release: true
+      pytorch_version: ""
```

Comment:
What's the purpose of adding these `""`?

Reply:
empty string is mainly used by nightly because we don't know the nightly version of pytorch before we start to run the workflow.
But, for release/rc-release, we would add a hard code value to it like `1.11.0` to enforce our dependency is fixed.